### PR TITLE
add yshrsmz keymaps

### PR DESCRIPTION
**Pull request by yshrsmz**
added my keymaps for erg42, helix, lets_split and nyquist.

### CODE REVIEW

File: keyboards/lets_split/keymaps/yshrsmz/config.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/lets_split/keymaps/yshrsmz/config.h]: If you could remove these three lines (including the include), replace it with `#pragma once` and remove the endif at the end?

File: keyboards/lets_split/keymaps/yshrsmz/rules.mk [hunk 2]
```diff
@@ -0,0 +1,5 @@
+AUTO_SHIFT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/lets_split/keymaps/yshrsmz/rules.mk]: Could you removethis block?

File: keyboards/nyquist/keymaps/yshrsmz/config.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/nyquist/keymaps/yshrsmz/config.h]: If you could remove these three lines (including the include), replace it with `#pragma once` and remove the endif at the end?

File: keyboards/nyquist/keymaps/yshrsmz/keymap.c [hunk 4]
```diff
@@ -0,0 +1,197 @@
+#include "nyquist.h"
```
Comment (drashna) [inline keyboards/nyquist/keymaps/yshrsmz/keymap.c]: Could you replace all of these includes with just `#include QMK_KEYBOARD_H`?

File: keyboards/nyquist/keymaps/yshrsmz/rules.mk [hunk 5]
```diff
@@ -0,0 +1,5 @@
+AUTO_SHIFT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/nyquist/keymaps/yshrsmz/rules.mk]: This block isn't needed, could you remove it?

**Comment by yshrsmz** [comment]
@drashna Hi, thanks for your review!
I've addressed your change requests, so please take another look.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**